packages/components/psammead-program-card/src/index.jsx

```diff
@@ -0,0 +1,169 @@
+import {
+  getLongPrimer,
+  getMinion,
+  getPica,
+} from '@bbc/gel-foundations/typography';
+import { Link } from '@bbc/psammead-story-promo';
+
+const programStateColors = {
+  live: {
+    backgroundColor: C_POSTBOX,
+    headerLabelColor: C_POSTBOX,
+    headerTextColor: C_EBON,
+    durationColor: C_WHITE,
+  },
+  next: {
+    backgroundColor: C_WHITE,
+    headerLabelColor: C_KINGFISHER,
+    headerTextColor: C_METAL,
+    durationColor: C_KINGFISHER,
+  },
+  onDemand: {
+    backgroundColor: C_EBON,
+    headerLabelColor: '',
+    headerTextColor: C_EBON,
+    durationColor: C_WHITE,
+  },
+};
+
+const CardWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+  background-color: ${C_WHITE};
+  position: relative;
+`;
+
+const TextWrapper = styled.div`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const HeadingWrapper = styled.h3`
+  ${({ service }) => service && getSerifRegular(service)};
+  ${({ script }) => script && getPica(script)};
+  color: ${({ headerTextColor }) => headerTextColor};
+  margin: 0; /* Reset */
+`;
+
+const LabelWrapper = styled.span`
+  ${({ service }) => service && getSansBold(service)};
+  ${({ script }) => script && getPica(script)};
+  text-transform: uppercase;
+  color: ${({ headerLabelColor }) => headerLabelColor};
+`;
+
+const DateWrapper = styled.div`
+  color: ${C_SHADOW};
+  padding: ${GEL_SPACING} 0;
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getPica(script)};
+`;
+
+const SummaryWrapper = styled(Paragraph)`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getLongPrimer(script)};
+  color: ${C_METAL};
+  padding-bottom: ${GEL_SPACING_DBL};
+`;
+
+const ButtonWrapper = styled.div`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getMinion(script)};
+  padding: ${GEL_SPACING} 0;
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  color: ${({ durationColor }) => durationColor};
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${({ durationColor }) => durationColor};
+    fill: currentColor;
+    width: 1.0625rem;
+    height: ${GEL_SPACING_DBL};
+    margin: 0;
+  }
+`;
+
+const DurationWrapper = styled.span`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const ProgramCard = ({
+  dir,
+  service,
+  script,
+  heading,
+  summary,
+  date,
+  duration,
+  state,
+  ctaLink,
+}) => (
+  <CardWrapper>
+    <Link href={ctaLink}>
+      <TextWrapper dir={dir}>
+        <HeadingWrapper
+          service={service}
+          script={script}
+          {...programStateColors[state]}
+        >
+          {state !== 'onDemand' && (
+            <LabelWrapper
```

Comment:
Hmm, we already have a LiveLabel in the story-promo package, that's also being used in the Bulletin component. The Link component from story-promo is also being used both in the story promo, bulletin, and in here. Should we extend the LiveLabel to include this Next state as well, and pull this shared functionality out of story promo as it's used in 3 places now?

Reply:
If we use `LiveLabel` from story-promo, we'll end up overriding all of it's styling since this card's Label uses a different set of colors, margins, and typography.